src/content/protocols/sky.md

```diff
@@ -0,0 +1,397 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "H", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "2025-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Sky is a stablecoin protocol allowing users to mint its USDS stablecoin through _Collateralized Debt Positions_ with a variety of crypto collateral assets. Sky is built on the Maker protocol, by the MakerDAO, to replace the DAI stablecoin and allow for the 1:1 conversion between DAI and USDS. USDS can further be staked for the yield-bearing sUSDS token. Various levels of control over the Sky protocol are exercised through an onchain governance system.
+
+# Ratings
+
+## Chain
+
+This review focuses on the Ethereum deployment of Sky, currently the only chain on which Sky protocol is deployed. Ethereum achieves a Low centralization risk score."
+
+> Chain score: Low
+
+## Upgradeability
+
+`USDS` and `sUSDS`, Sky's stablecoin and its corresponding yield-bearing version, are upgradeable contracts through Sky governance proposals. Updating those contracts could change the entire logic of those tokens and may incur a loss of funds for users.
+
+Many parameters in the Sky Protocol can be changed through governance proposals and may also incur loss of funds, loss of unclaimed yield, as well as materially impact the protocol's performance. Example actions are forced liquidations, which would result in loss of user funds, creating unbacked debt, which could endanger the protocol's stability, or pausing the contracts, which could trap user funds for an undetermined amount of time.
```

Comment:
"Critical parameters in the Sky protocol can be changed through _Sky Governance_ proposals. Unwanted updates to these parameters can result in the loss of funds, loss of unclaimed yield, or otherwise materially impact the expected protocol performance.

"Sky protocol"